source/utilities.tex

```diff
@@ -21135,3 +21135,719 @@
+\begin{itemdecl}
+const_reference operator[](size_type frame_no) const;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{frame_no < size()}.
+
+\pnum
+\returns
+\tcode{frames_[frame_no]}.
+
+\pnum
+\throws
+Nothing.
+\end{itemdescr}
+
+\indexlibrarymember{at}{basic_stacktrace}%
+\begin{itemdecl}
+const_reference at(size_type frame_no) const;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{frames_[frame_no]}.
+
+\pnum
+\throws
+\tcode{out_of_range} if \tcode{frame_no >= size()}.
+\end{itemdescr}
+
+\rSec3[stacktrace.basic.cmp]{Comparisons}
+
+\indexlibrarymember{operator==}{basic_stacktrace}%
+\begin{itemdecl}
+template<class Allocator2>
+friend bool operator==(const basic_stacktrace& x, const basic_stacktrace<Allocator2>& y) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{equal(x.begin(), x.end(), y.begin(), y.end())}
+\end{itemdescr}
+
+\indexlibrarymember{operator<=>}{basic_stacktrace}%
+\begin{itemdecl}
+template<class Allocator2>
+friend strong_ordering
+  operator<=>(const basic_stacktrace& x, const basic_stacktrace<Allocator2>& y) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{x.size() <=> y.size()} if \tcode{x.size() != y.size()};
+\tcode{lexicographical_compare_3way(\newline
+x.begin(), x.end(), y.begin(), y.end())}
+otherwise.
+\end{itemdescr}
+
+\rSec3[stacktrace.basic.mod]{Modifiers}
+
+\indexlibrarymember{swap}{basic_stacktrace}%
+\begin{itemdecl}
+void swap(basic_stacktrace& other)
+  noexcept(allocator_traits<Allocator>::propagate_on_container_swap::value ||
+    allocator_traits<Allocator>::is_always_equal::value);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Exchanges the contents of \tcode{*this} and \tcode{other}.
+\end{itemdescr}
+
+\rSec3[stacktrace.basic.nonmem]{Non-member functions}
+
+\indexlibrarymember{swap}{basic_stacktrace}%
+\begin{itemdecl}
+template<class Allocator>
+void swap(basic_stacktrace<Allocator>& a, basic_stacktrace<Allocator>& b)
+  noexcept(noexcept(a.swap(b)));
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to \tcode{a.swap(b)}.
+\end{itemdescr}
+
+\indexlibrarymember{to_string}{basic_stacktrace}%
+\begin{itemdecl}
+string to_string(const stacktrace_entry& f);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+A string with a description of \tcode{f}.
+
+\recommended
+The description should provide information about the contained evaluation,
+including information from
+\tcode{f.source_file()} and \tcode{f.source_line()}.
+\end{itemdescr}
+
+\indexlibrarymember{to_string}{basic_stacktrace}%
+\begin{itemdecl}
+template<class Allocator>
+string to_string(const basic_stacktrace<Allocator>& st);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+A string with a description of \tcode{st}.
+\begin{note}
+The number of lines is not guaranteed to be equal to \tcode{st.size()}.
+\end{note}
+\end{itemdescr}
+
+\indexlibrarymember{operator<<}{basic_stacktrace}%
```

Comment:
Is it intentional that `stacktrace_entry` doesn't get its own? And does this indexes both overloads?

Reply:
Having seen your fixup of indexing:
```suggestion
\indexlibrarymember{operator<<}{stacktrace_entry}%
```

Reply:
fixed